app/views/avo/actions/show.html.erb

```diff
@@ -51,7 +51,7 @@
             size: :sm,
             data: {
               target: :submit_action,
-            } do %>
+            }, autofocus: (@fields.count == 2 ? true : false)  do %>
```

Comment:
```suggestion
            }, autofocus: @fields.empty?  do %>
```

Lets autofocus the run button when fields are empty instead when there are exactly 2 fields

Reply:
ok, I will update it.